frontend/packages/metal3-plugin/src/components/host-detail.tsx

```diff
@@ -218,7 +244,21 @@ export const BaremetalHostDetailPage: React.FC<BaremetalHostDetailPageProps> = (
         navFactory.events(ResourceEventStream),
       ]}
       kind={referenceForModel(BaremetalHostModel)}
-      resources={[machinesResource, nodesResource]}
+      resources={resources}
+      menuActions={menuActionsCreator}
+      customData={{ hasNodeMaintenanceCapability }}
     />
   );
 };
+
+const hostPageStateToProps = ({ k8s }) => ({
+  hasNodeMaintenanceCapability: !!k8s.getIn([
+    'RESOURCES',
+    'models',
+    referenceForModel(NodeMaintenanceModel),
+  ]),
+});
+
+export const BaremetalHostDetailPageConnected = connect(hostPageStateToProps)(
+  BaremetalHostDetailPage,
+);
```

Comment:
using `Connected` convention due to `no-underscore-dangle` rule being enforced  by eslint

Reply:
> `no-underscore-dangle` rule being enforced by eslint

Individual plugins can always disable or tweak rules in their `packages/<pkg>/.eslintrc(.js)`, assuming it extends the parent `packages/.eslintrc.js`.

The `Connected` suffix convention comes from migrated kubevirt/web-ui code. Existing code in `public` uses a trailing underscore.

On the other hand, DevConsole plugin has adopted "default export per module" approach, e.g. `packages/dev-console/src/components/AddPage.tsx` which has `export default AddPage`.

Based on earlier discussions, all packages should converge to convention that's already used in DevConsole plugin (and many other React projects, including `kubevirt-web-ui-components`).

Reply:
TL;DR I'm OK with `Connected` convention for now, but we should adapt & unify code in the long term.

Reply:
leaving `Connected` for now